### DISCUSSION TITLE
Request to add openGauss database as data source

### DISCUSSION THREAD

**Discussion by travelliu** [grafana/grafana#51663] [Data Sources]
openGauss is an open source relational database management system that is released with the Mulan PSL v2. with the kernel built on Huawei's years of experience in the database field and continuously provides competitive features tailored to enterprise-grade scenarios.

The SHA256 password authentication method between openGauss and PostgreSQL is different. The general PG driver cannot be used to connect. Can it be adapted?

https://opengauss.org/en/

https://gitee.com/opengauss/openGauss-connector-go-pq

docker image: docker pull enmotech/opengauss

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
